cap/modules/experiments/utils/common.py

```diff
@@ -91,7 +91,7 @@ def generate(url):
     return generate(url)
 
 
-def recreate_es_index_from_source(alias, source, mapping=None):
+def recreate_es_index_from_source(alias, source, mapping=None, settings={}):
```

Comment:
tsk tsk, better not ^^
https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html